lib/serializer.ts

```diff
@@ -435,8 +457,30 @@ export class Serializer {
         }
       }
 
-      for (const key in modelProps) {
-        if (modelProps.hasOwnProperty(key)) {
+      for (const key of Object.keys(modelProps)) {
+        const propertyMapper = modelProps[key];
+        let propertyObjectName = objectName;
+        if (propertyMapper.serializedName !== "") {
+          propertyObjectName = objectName + "." + modelProps[key].serializedName;
```

Comment:
modelProps[key] -> propertyMapper (just for consistency)

Reply:
Done